llvm/test/CodeGen/RISCV/stack-guard-global.ll

```diff
@@ -0,0 +1,50 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -verify-machineinstrs < %s | \
+; RUN: FileCheck %s
```

Comment:
The line above isn't (currently) long enough to warrant wrapping, but if you do you need to indent it like normal code so it's obviously a continuation line

Reply:
Thanks. After moving the triple into the RUN line, it's long enough that wrapping looks cleaner. I found a couple of examples using indentation on the continuation line to follow.